scorecard.js

```diff
@@ -0,0 +1,21 @@
+const Frame = require('./frame');
+
+class Scorecard {
+  constructor() {
+    this.frames = []
+  }
+
+  inputFrame(rolls) { //rolls will be e.g. [2,3]
+    const frame = new Frame(rolls); // would it be better to inject this in? If I inject it, would I then need to use a double in the tests?
```

Comment:
You can inject the Frame class in your constructor, and then reuse it.
You would then need to mock that class in some way.
```js
constructor(frameClass = Frame) {
    this.frames = []
    this.frameClass = frameClass
  }
  
  // then
  const frame = new this.frameClass(rolls)
```